fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/NodePropertiesTools.java

```diff
@@ -234,11 +234,6 @@ public void removeNodeProperty(final Node node, final String propertyName, final
                         LOGGER.debug("Removing value '{}' from property '{}'", strValueToRemove, propertyName);
                         property.setValue(newValues);
                     }
-                } else {
-                    LOGGER.debug("Value not removed from property name '{}' (value '{}')", propertyName,
```

Comment:
I think there is some value to leaving the debug statement in here, if that part doesn't cause issues.  That way it's really clear that the function failed to remove the triple(s).  Else it looks like it will be silent failure, where even turning on debug won't help.